internal/constants/constants.go

```diff
@@ -172,7 +172,7 @@ const (
 	// TerraformIAMUpgradeBackupDir is the directory name being used to backup the pre-upgrade state of iam in an upgrade.
 	TerraformIAMUpgradeBackupDir = "terraform-iam-backup"
 	// TerraformEmbeddedDir is the name of the base directory embedded in the CLI binary containing the Terraform files.
-	TerraformEmbeddedDir = "terraform"
+	TerraformEmbeddedDir = "infrastructure"
```

Comment:
why change this?

Reply:
because @3u13r wanted it :) `terraform/terraform` is a confusing naming to refer to the relevant infra files for the CLI